alpine3.15.1/Dockerfile

```diff
@@ -2,35 +2,36 @@ ARG BUILD_FROM=alpine:3.15.1
 FROM ${BUILD_FROM}
 
 RUN apk add --no-cache --virtual .build-deps \
+                autoconf \
+                automake \
+                bash \
+                bsd-compat-headers \
                 ca-certificates \
-                zip \
+                coreutils \
+                curl \
                 dpkg-dev dpkg \
-                gcc \
+                elixir \
```

Comment:
This one shouldn't be needed, because we download and compile a specific version [in another step](https://github.com/emqx/emqx-builder/blob/931ed0ce0b17105586848b7771904b7a4f9732e8/Dockerfile#L11).

Reply:
I've just seen that it was present before, but that's probably a mistake :see_no_evil:

Reply:
Removed [411abd4](https://github.com/emqx/emqx-builder/pull/29/commits/411abd4bc2cfb25731af5d8890b4b13e98f10323)